src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -667,6 +667,16 @@
     crafted: true
 """
 
+helps['functionapp function list'] = """
+type: command
+short-summary: List functions in a function app.
+examples:
+  - name: List functions.
+    text: >
+        az functionapp function list -g MyResourceGroup -n MyFunctionAppName
+    crafted: true
```

Comment:
For the manually added example, it is not necessary to add the `crafted: true` label
```suggestion
```

Reply:
Any update for this comment?

Reply:
Thank you for your feedback @zhoxing-ms I just made the changes